db/migrate/20170530102506_add_deleted_to_containers_tables.rb

```diff
@@ -0,0 +1,25 @@
+class AddDeletedToContainersTables < ActiveRecord::Migration[5.0]
+  def change
+    add_column :container_definitions, :deleted, :boolean, :default => false, :null => false
+    add_column :container_groups, :deleted, :boolean, :default => false, :null => false
+    add_column :container_images, :deleted, :boolean, :default => false, :null => false
+    add_column :container_projects, :deleted, :boolean, :default => false, :null => false
+    add_column :containers, :deleted, :boolean, :default => false, :null => false
```

Comment:
Right, we use a DB level replication, that argument is not valid, but good to know the past reasons. :-)

So saying that, what are the other concrete blockers? Given we want to design it right, this should be the correct way. So I would like to avoid bad DB design, unless there are real concerns.

Btw. I can't think of any reason that would require a later change in this column. Deleted true/false should be the end state.